packages/core/package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "@viamrobotics/prime-core",
-  "version": "0.0.8",
+  "version": "0.0.9",
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
+    "prepare": "svelte-kit sync",
     "dev": "vite dev",
     "build": "vite build && pnpm run package",
     "preview": "vite preview",
     "package": "svelte-kit sync && svelte-package && publint",
-    "prepublishOnly": "pnpm run package",
```

Comment:
Note: `prepublishOnly` implicitly runs before `publish`, which means we were implicitly rebuilding the project during publish, even though we were pulling down the built artifacts. Fun stuff